torchaudio/functional.py

```diff
@@ -290,18 +290,19 @@ def amplitude_to_DB(x, multiplier, amin, db_multiplier, top_db=None):
 
 
 @torch.jit.script
-def create_fb_matrix(n_freqs, f_min, f_max, n_mels):
-    # type: (int, float, float, int) -> Tensor
+def create_fb_matrix(n_freqs, f_min, f_max, n_mels, sample_rate):
+    # type: (int, float, float, int, int) -> Tensor
     r"""
-    create_fb_matrix(n_freqs, f_min, f_max, n_mels)
+    create_fb_matrix(n_freqs, f_min, f_max, n_mels, sample_rate)
 
     Create a frequency bin conversion matrix.
 
     Args:
         n_freqs (int): Number of frequencies to highlight/apply
-        f_min (float): Minimum frequency
-        f_max (float): Maximum frequency
+        f_min (float): Minimum frequency (Hz)
+        f_max (float): Maximum frequency (Hz)
```

Comment:
Ths units depend on the units of `sample_rate` which need not be in Hz, right?

Reply:
You are correct, that's indeed what we use in the readme. :) WIth/without is fine by me.

Reply:
@Vincent - I think it does have to be in Hz.  In the create_fb_matrix function there are hardcoded constants like 2595 which are conversions from HZ to Mel Scale.